docs/tutorials/data_management_interactive.md

```diff
@@ -53,12 +53,14 @@ Now, update the instatiation of the datetime formatter to load data from the blo
 locale is Chakma:
 
 ```rust
-// new imports
+// At the top of the file:
 use icu::locid::locale;
 use icu_provider_blob::BlobDataProvider;
 
+// Just below the imports (fill in the path):
 const CCP_BLOB_PATH: &str = "<absolute path to ccp.blob>";
 
+// In the main() function:
```

Comment:
In the main function is not specific enough. I say above "update the instatiation of the datetime formatter", that should be enough
```suggestion
```